contracts/vault/CallWhitelistApprovals.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.11;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
```

Comment:
The ownable.sol import is not needed here since it is inherited in CallWhitelist and CallWhitelist is inherited here.